rdt-app/src/main/java/io/ona/rdt_app/util/RDTJsonFormUtils.java

```diff
@@ -76,21 +80,23 @@ protected ProfileImage doInBackground(Void... voids) {
                         profileImage.setAnmId(providerId);
                         profileImage.setEntityID(entityId);
                         profileImage.setFilepath(absoluteFileName);
-                        profileImage.setFilecategory(RDT_IMAGE);
+                        profileImage.setFilecategory(MULTI_VERSION);
                         profileImage.setSyncStatus(ImageRepository.TYPE_Unsynced);
                         ImageRepository imageRepo = RDTApplication.getInstance().getContext().imageRepository();
                         imageRepo.add(profileImage);
 
-//                        saveImageToGallery(context, image); // todo: only enable this in debug apks
+                        if (BuildConfig.SAVE_IMAGES_TO_GALLERY) {
+                            saveImageToGallery(context, image);
+                        }
                     }
                 } catch (FileNotFoundException e) {
-                    Log.e(TAG, "Failed to save static image to disk");
+                    Log.e(TAG, e.getStackTrace().toString());
                 } finally {
                     if (os != null) {
                         try {
                             os.close();
                         } catch (IOException e) {
-                            Log.e(TAG, "Failed to close static images output stream after attempting to write image");
+                            Log.e(TAG, e.getStackTrace().toString());
```

Comment:
Use Timber for logging, or just invoke Log.e(Tag, e.getMessage() or custom message as previous ,e);

Reply:
Done.